external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -309,4 +309,21 @@
 
   </Target>
 
+  <Target Name="AddRemoteExecutorLib"
+          Condition="'$(TargetsNetCoreApp)' == 'true' or '$(TargetsNetFx)' == 'true'"
+          BeforeTargets="ResolveReferences">
+
+    <Error Condition="!Exists('$(PackagesDir)$(MicrosoftDotNetRemoteExecutorPackage)\$(MicrosoftDotNetRemoteExecutorPackageVersion)\lib\netstandard2.0\Microsoft.DotNet.RemoteExecutor.dll')"
+           Text="Error: looks the package $(PackagesDir)$(MicrosoftDotNetRemoteExecutorPackage)\$(MicrosoftDotNetRemoteExecutorPackageVersion) not restored or missing Microsoft.DotNet.RemoteExecutor.dll." />
+
+    <ItemGroup>
+      <ReferenceCopyLocalPaths Include="$(PackagesDir)$(MicrosoftDotNetRemoteExecutorPackage)\$(MicrosoftDotNetRemoteExecutorPackageVersion)\lib\netstandard2.0\*.*">
```

Comment:
These items are already included so you're depending on item ordering for removing ones already resolved.  Can you make sure to remove them first, or use excludeassets?